### ISSUE TITLE
Sidebar display width collapse issue

### ISSUE THREAD

**Issue by Amparose** [careteditor/caret#593]
Checked the beta and the issue is still present: the width of the sidebar is not user-controllable and behaves somewhat poorly at widths around 940px wide.

**Comment by Amparose** [comment]
Hmmm is there also no way to completely hide the sidebar now? This bug is irritating but at least in the non-beta the sidebar could be hidden altogether. Now I'm stuck looking at this:

![screenshot 2018-01-04 15 19 42](https://user-images.githubusercontent.com/949192/34567474-bfe6b962-f162-11e7-8d40-e49c46078488.png)

**Comment by crankygeologist** [comment]
I agree that the sidebar behavior is not very intuitive. You have to mess with the physical width of the main note window and tweak the line width in the settings. Currently, you can show/hide the files list with a keyboard shortcut, but not the tab list. You must right-click on the tab list and choose "Close all tabs" or continue pressing "Cmd W" until all of the open tab files are closed.

**Comment by Amparose** [comment]
Yeah... tabs should be "tabs" like at the top of the window. I really don't like this design idea of a weird, ugly, list of documents that are kinda were open, permanently cluttering the side of the interface. It's also confusing that I can "close all tabs" but current document is still open and it just all feels weird and not like "tabs".

The sidebar tabs idea works in VSCode (making a grand assumption that this is partly inspiration) but: its quasi-optional in that it can be collapsed, proper "tabs" exist at the top of the editor to represent the current/other open documents, and the entire sidebar can be hidden.

Hate to bring it up (although I bought Caret) but Typora handles this much better.

**Comment by erusev** [comment]
We'll improve how tabs work in the next beta release, they'll still be on the left but they should be more intuitive to work with.

**Comment by erusev** [comment]
Ok, beta 7 is now released and tabs are at the top :)

**Comment by Amparose** [comment]
Yay! Really appreciate the "Show Nothing" option and I like the tabs up top now along with the fact that Caret doesn't open to new tab with CMD T. Much cleaner. The sidebar collapse issue is still there but now that the sidebar can be hidden it has less impact.

**Comment by crankygeologist** [comment]
I also like the new tabs-on-top feature. I discovered you can navigate to specific tabs with Cmd 1, Cmd 2, Cmd 3, etc. Nice! Maybe there could be a mouse-draggable divider to dynamically change the width of the sidebar.

**Comment by erusev** [comment]
> Maybe there could be a mouse-draggable divider to dynamically change the width of the sidebar.

We're working on something like that.

> along with the fact that Caret doesn't open to new tab with CMD T

It's not that obvious right now, but if you need to open a file in a new tab from CmdT, you can use CmdEnter.

**Comment by sheldonhull** [comment]
I came across this post as a new test user. Sidebar kept disappearing even when maximized. I'm guessing there is no "pin" sidebar option so I can leave it open at this time, right?
